nav2_util/test/test_geometry_utils.cpp

```diff
@@ -49,3 +51,24 @@ TEST(GeometryUtils, euclidean_distance_pose)
 
   ASSERT_NEAR(euclidean_distance(pose1, pose2), 10.24695, 1e-5);
 }
+
+TEST(GeometryUtils, calculate_path_length)
```

Comment:
Can you add a test where you generate a circle and test that the circle perimeter is within reasonable tolerances to the actual radius? That way its not just a strightline test

Reply:
Great idea! Just added it and the results are within 10 cm of the circumference given a 1 deg discretization of the perimeter.